kalite/testing/base_environment.py

```diff
@@ -11,20 +11,19 @@
 from urlparse import urljoin
 from django.contrib.auth.models import User
 
-from kalite.testing.behave_helpers import login_as_admin, login_as_coach, logout
+from kalite.testing.behave_helpers import login_as_admin, login_as_coach, logout, login_as_learner
 
 def before_all(context):
-    browser = context.browser = webdriver.Firefox()
-    # ensure the window is reasonably sized.
-    browser.set_window_size(1024, 768)
+    pass
 
 def after_all(context):
-    try:
-        context.browser.quit()
-    except CannotSendRequest:
-        pass
+    pass
 
 def before_feature(context, feature):
+    browser = context.browser = webdriver.Firefox()
+    # ensure the window is reasonably sized.
+    browser.set_window_size(1024, 768)
```

Comment:
Just a few thoughts:
* What happens if the content isn't available? I would still expect the page to load, but I have encountered issues with tests that passed locally but failed on travis because I had content where travis doesn't.
* Which long-running functions are triggered first by the learn page __views__, if any? Could something be causing the learn page to hang because Django is taking a long time to come up with a response?